test/testutils.js

```diff
@@ -10,6 +10,10 @@ if (window.__AXE_EXTENSION__) {
 /*eslint indent: 0*/
 var testUtils = {};
 
+var fixture = document.createElement('div');
```

Comment:
We don't use the `runner.tmpl` file anymore so need to create the fixture element

Reply:
You could use something like [karma fixture](https://www.npmjs.com/package/karma-fixture). This will help reduce the number of changes that you have to do to make integration tests work.